crates/aptos/src/move_tool/stored_package.rs

```diff
@@ -129,7 +129,13 @@ impl<'a> CachedPackageMetadata<'a> {
         let sources_dir = path.join(CompiledPackageLayout::Sources.path());
         fs::create_dir_all(&sources_dir)?;
         for module in &self.metadata.modules {
-            let source = unzip_metadata_str(&module.source)?;
+            let source = match module.source.is_empty() {
+                true => {
+                    println!("module without code: {}", module.name);
+                    "".into()
+                }
```

Comment:
probably should just make this an error.

but good find.

Reply:
> 
Thanks review.

If someone deploys the contract with `aptos move publish --included-artifacts none`, then the `source field` is indeed empty. `move download` should not treat it as an exception and report an error.

Reply:
I guess this is good enough for now then... @gregnazario